docs/openapi.md

```diff
@@ -20,7 +20,7 @@ gen.addSchema("v1.<YOUR_PAYLOAD>", &payloads.<YOUR_PAYLOAD>{})
 ```
 3. use the `addSchema` for registering new errors payloads if needed
 
-### How to add new new path
+### How to add a new path
 Edit `/cmd/spec/paths.yaml` for adding the new route manually
```

Comment:
it is path not paths :)

Reply:
the file paths.yaml does not exist, it is path.yaml :)

Reply:
fixed :)